docs/cpp/configure-intellisense.md

```diff
@@ -0,0 +1,192 @@
+---
+Order: 13
+Area: cpp
+TOCTitle: Configure IntelliSense
+ContentId: bf494c65-12b4-4506-ab6c-1fad76d7ccf1
+PageTitle: Configure C/C++ IntelliSense
+DateApproved: 10/31/2023
+MetaDescription: Configure Visual Studio Code IntelliSense in the C/C++ Extension
+---
+# Configure C/C++ IntelliSense
+
+This article is about configuring the C/C++ extension to provide C++ specific [IntelliSense](/docs/editor/intellisense.md) suggestions in VS Code. IntelliSense is a helpful tool built into VS Code that provides various code editing features that help you code faster and more efficiently. For example, code completion, parameter info, syntax highlighting, code actions (light bulbs), and member lists are all generated using IntelliSense.
+
+C/C++ IntelliSense only requires a C/C++ compiler to be installed on your system. The C/C++ compiler provides C++ specific information to IntelliSense, such as the locations of system include paths and other settings. For project level configurations, reference [Project level IntelliSense Configuration](/docs/cpp/configure-intellisense.md#_Project-Level-IntelliSense-Configuration) section.
```

Comment:
/docs/cpp/configure-intellisense.md#project-level-intellisense-configuration
use lower case and don't include leading underscore